test/data.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  INVALID_QUERY_ERR: 'invalid_query_err',
+  EVERYDAY_OBJECT_ID: '598e23679fd1a805a5c09275',
+  RANDOM_MONGO_ID: '5d0966ba1eff390eb5d9e2c2',
+  DETECTOR_ZIP: `${__dirname}/test-files/detector.zip`,
+  DOG_FILE: `${__dirname}/test-files/dog.jpg`,
+  CAT_FILE: `${__dirname}/test-files/cat.jpg`,
+  DOG_URL: 'https://matroid-web.s3.amazonaws.com/test/python-client/dog.jpg',
+  CAT_URL:
+    'https://matroid-web.s3.amazonaws.com/test/python-client/tesla-cat.jpg',
+  UPLOAD_PB_FILE: `${__dirname}/test-files/upload-detector.pb`,
+  UPLOAD_LABEL_FILE: `${__dirname}/test-files/upload-detector-label.txt`,
+  YOUTUBE_VID_URL: 'https://www.youtube.com/watch?v=nDvh3upNJmA',
```

Comment:
Let's use an S3 url instead like we do in selenium. Youtube has been pretty consistently blocking us.